modape/scripts/modis_window.py

```diff
@@ -159,6 +160,9 @@ def cli(src: str,
     if round_int is not None:
         round_int = round_int * -1
 
+    if target_srs.lower() == "none":
```

Comment:
The user can basically specify `--target-srs None` or `none` or `NoNe` ... and it'll be evaluated to `None`.

This way we can keep the convenient default of 4326, while enabling using the existing srs without specifying the actual reference